libs/gltfio/src/AssetLoader.cpp

```diff
@@ -202,14 +205,22 @@ class MaterialInstanceCache {
 };
 
 struct FAssetLoader : public AssetLoader {
-    FAssetLoader(const AssetConfiguration& config) :
+    FAssetLoader(AssetConfiguration const& config) :
             mEntityManager(config.entities ? *config.entities : EntityManager::get()),
             mRenderableManager(config.engine->getRenderableManager()),
             mNameManager(config.names),
             mTransformManager(config.engine->getTransformManager()),
             mMaterials(*config.materials),
             mEngine(*config.engine),
-            mDefaultNodeName(config.defaultNodeName) {}
+            mDefaultNodeName(config.defaultNodeName) {
+        if (config.ext) {
+#if defined(__ANDROID__) || defined(IOS) || defined(__EMSCRIPTEN__)
+            PANIC_PRECONDITION("Extend asset loading is not supported on this platform");
+#endif
```

Comment:
Is there a way to query if it's supported?

Reply:
moved this to a static method so that client can use the method to check for support.